src/thing/get.ts

```diff
@@ -281,6 +281,43 @@ export function getStringWithLocaleAll(
   return matchingQuads.map((quad) => quad.object.value);
 }
 
+/**
+ * Retrieves all string literals for the specified property from the specified
+ * Thing.
+ *
+ * NOTE: Assumes you also want non-locale string literals too (i.e. literals
```

Comment:
Just for consistency with other comments in the codebase.

```suggestion
 * Note: Assumes you also want non-locale string literals too (i.e. literals
```